Formula/g/genders.rb

```diff
@@ -33,13 +33,14 @@ class Genders < Formula
 
```

Comment:
```suggestion

  on_linux do
    depends_on "python-setuptools" => :build
  end

```

[`src/extensions/python/Makefile.am`](https://github.com/chaos/genders/blob/2824c2c0f7e586dbf1e9a4daa3b0c5471f2da631/src/extensions/python/Makefile.am#L7) uses `distutils`, so it looks like we need Python `setuputils` on Linux: `ModuleNotFoundError: No module named 'distutils'`. Not sure if this suggestion is the correct way to go about it but there's some prior art in other formulae, at least.